.travis.yml

```diff
@@ -25,6 +25,7 @@ before_install:
 #  - sudo apt-get install libcr-dev mpich2
 #  - sudo apt-get install mpich libmpich-dev # libhdf5-mpich-dev
   - sudo apt-get install libopenmpi-dev openmpi-bin # libhdf5-openmpi-dev
+  - sudo apt-get install default-jre
```

Comment:
One could avoid installing a full JRE by just loading `bash <(curl -Ls https://coverage.codacy.com/get.sh)` directly from codacy, see https://github.com/pyccel/fffi/blob/master/.travis.yml

Reply:
Interesting
I added that line because when I ran `bash <(curl -Ls https://coverage.codacy.com/get.sh)` locally on my computer it complained that I had not installed java.
That line was therefore necessary to satisfy the requirement : Java JRE 8 and higher
stated on https://github.com/codacy/codacy-coverage-reporter
Presumably travis already has a version of java installed somewhere which allows you to remove this line.

Do you think it is better to remove this line to avoid installing unnecessary packages or to leave it in case travis changes its internal packages at some point?

Reply:
Oh, in that case could be that really travis has it. Then I would leave  line inside, as it doesn't cost anything. For some reason I added also `apt install jq` in fffi, as this was mentioned somewhere - but it's not needed. So everything OK also here!